operations/research-and-development/product/development-process/new-bug-tickets/bug-priority-guidelines.md

```diff
@@ -0,0 +1,20 @@
+# Priority Levels for Tickets
+
+|   **Priority**   |                                      **Definition**                                      |                                                      **Example Issues**                                                     |    **Target Resolution**   | **Release Guidelines**                              |
+|:----------------:|:----------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|:--------------------------:|-----------------------------------------------------|
+| **High**         | **A customer-reported issue or blocker that prevents or severely delays a user’s goal.** | App crash, data loss, broken website forms, broken authentication flow                                                      | Fixed within 30 days       | Fix can be committed after T-10 Code Freeze         |
+| **Medium**       | **A frustrating friction point or an issue so “cringey” it erodes trust.**               | Documentation gaps, misleading error text or guidance, complex UX, inconsistent user interactions, functional regressions   | Fixed within 60 days       | Fix can be committed after T-19 Code Complete       |
+| **Low**          | **An inconsistency that affects polish.**                                                | UI misalignment, unclear labels, minor documentation errors, cosmetic regressions                                           | Fixed within 90 days       | Fix can only be committed before T-19 Code Complete |
+
+
+## Assigning priority levels
+
+Priority levels on Jira tickets should be filled in by the ticket reporter using the **Priority** field. If the field is left blank, the triage team will assign Priority based on the guidelines above. 
```

Comment:
New tickets will get the "Medium" priority by default, so no new tickets have a blank priority.